README.md

```diff
@@ -52,7 +52,47 @@ Generate team
 
 #### Configuring GatorGrade Checks
 
-Input team
+There are multiple customizable options with gatorgrade!
+Gatorgrader checks are able to run within a specific file context
+and without any file path.
+
+To run a check within the context of a file path,
+please be sure to include the path to the file above description,
+along with a description of the check and the options.
+See example below for reference.
```

Comment:
It would be super neat if all of the mentions of actual key names could be formatted as code! For example, this could be: 
> To configure a check to be run within the context of a file path, please be sure to include the path to the file above the start of a check. Then, you can define a description for the check by using the `description` key.
And so on and so forth 🙂